src/app-layout/__integ__/awsui-applayout.test.ts

```diff
@@ -167,11 +167,18 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as Theme[])('%s', theme
   test(
     'does not render notifications slot when it is empty',
```

Comment:
We already had a test which was supposed to catch the issue, but it was asserting size of the wrong element. Refactored the test. It is now more strict and may require an update even on unrelated design changes, but this is still better than silently passing

Reply:
This test relied on hard-coded numbers which are different on local and CI environment.

Let's keep the current test and rely on screenshot tests to capture this regression